docs/gen/config/controller/config-swagger.yaml

```diff
@@ -1246,6 +1261,16 @@ info:
     description: Aperture Controller
     title: Controller
 paths:
+    /agent_info:
```

Comment:
no idea :D

Reply:
then why is this change here? did you do it :D ?

Reply:
some hook did this. Will check what's going on

Reply:
Oh, I get it. It's because controller now depends on election package which depends on agent info (even though it's not actually used in controller). Will need to split packages.